src/bolos/cx_utils.h

```diff
@@ -3,6 +3,10 @@
 #include <stdint.h>
 #include <unistd.h>
 
+#if INTPTR_MAX == INT64_MAX
```

Comment:
I was wondering whether any other project was doing such a check, and actually Musl is doing this :) https://git.musl-libc.org/cgit/musl/tree/include/stdint.h?h=v1.2.3#n105

... but it uses the unsigned type instead: `#if UINTPTR_MAX == UINT64_MAX`. I believe it makes sense to use unsigned types instead (as "pointers" are usually considered unsigned), unless there is a strong argument for using `INTPTR_MAX == INT64_MAX` that I missed.

TL;DR: LGTM, and feel free to change it to `#if UINTPTR_MAX == UINT64_MAX` if you want

Reply:
Thanks! (I update macros to unsigned version.)